server/channels/api4/post_test.go

```diff
@@ -124,56 +123,64 @@ func TestCreatePost(t *testing.T) {
 	})
 
 	t.Run("Create posts without the USE_CHANNEL_MENTIONS Permission - returns ephemeral message with mentions and no ephemeral message without mentions", func(t *testing.T) {
-		WebSocketClient, err := th.CreateWebSocketClient()
-		WebSocketClient.Listen()
+		wsClient, err := th.CreateWebSocketClient()
```

Comment:
We have to use the `th.CreateConnectedWebSocketClient` which @streamer45  recently merged.

Reply:
Thanks for letting me know that that helper exists! I've changed the base branch of this PR to point to https://github.com/mattermost/mattermost/pull/29893.